lib/utils/config/getServerlessDir.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const path = require('path');
+const os = require('os');
+
+module.exports = function getServerlessDir() {
```

Comment:
Could we move this code to the `Utils` class in `lib/classes/Utils.js` since that's the central place for code like this.